ui/raidboss/data/06-ew/trial/endsinger.txt

```diff
@@ -0,0 +1,107 @@
+132.2 "Elegeia Unforgotten" sync / 1[56]:[^:]*:The Endsinger:6682:/
+158.4 "Fatalism" sync / 1[56]:[^:]*:The Endsinger:6632:/
+185.6 "Elegeia Unforgotten" sync / 1[56]:[^:]*:The Endsinger:6682:/
+211.7 "Fatalism" sync / 1[56]:[^:]*:The Endsinger:6632:/
+223.9 "Galaxias" sync / 1[56]:[^:]*:The Endsinger:6C69:/
+238.1 "Ekstasis" sync / 1[56]:[^:]*:The Endsinger:6645:/
+241.9 "Epigonoi" sync / 1[56]:[^:]*:The Endsinger:6646:/
+244.4 "--sync--" sync / 1[56]:[^:]*:The Endsinger:6647:/
+262.1 "--sync--" sync / 1[56]:[^:]*:The Endsinger:6692:/
+265.1 "--untargetable--"
+274.4 "Interstellar" sync / 1[56]:[^:]*:The Endsinger:664D:/
+276.6 "Nemesis" sync / 1[56]:[^:]*:The Endsinger:664E:/
+282.6 "--targetable--"
+285.7 "Aporrhoia" sync / 1[56]:[^:]*:The Endsinger:663D:/
+295.3 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:663E:/
+313.1 "Elegeia Unforgotten" sync / 1[56]:[^:]*:The Endsinger:6682:/
+332.3 "Elegeia Unforgotten" sync / 1[56]:[^:]*:The Endsinger:6682:/
+357.5 "Fatalism" sync / 1[56]:[^:]*:The Endsinger:6632:/
+369.7 "Galaxias" sync / 1[56]:[^:]*:The Endsinger:6C69:/
+375.9 "Fatalism" sync / 1[56]:[^:]*:The Endsinger:6632:/
+388.0 "Nemesis" sync / 1[56]:[^:]*:The Endsinger:6654:/
+401.2 "--sync--" sync / 1[56]:[^:]*:The Endsinger:664A:/
+407.2 "Death's Embrace" sync / 1[56]:[^:]*:The Endsinger:6649:/
+417.3 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:6644:/
+426.4 "Hubris" sync / 1[56]:[^:]*:The Endsinger:6652:/
+436.6 "Aporrhoia" sync / 1[56]:[^:]*:The Endsinger:663D:/
+446.2 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:663E:/
+450.8 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:6644:/
+464.0 "Elegeia" sync / 1[56]:[^:]*:The Endsinger:662C:/
+483.2 "Nemesis" sync / 1[56]:[^:]*:The Endsinger:6654:/
+490.3 "--untargetable--"
+499.6 "Interstellar" sync / 1[56]:[^:]*:The Endsinger:664D:/
+501.8 "Nemesis" sync / 1[56]:[^:]*:The Endsinger:664E:/
+507.8 "--targetable--"
+513.9 "--sync--" sync / 1[56]:[^:]*:The Endsinger:664A:/
+519.9 "Death's Embrace" sync / 1[56]:[^:]*:The Endsinger:6649:/
+539.2 "Hubris" sync / 1[56]:[^:]*:The Endsinger:6652:/
+549.4 "Aporrhoia" sync / 1[56]:[^:]*:The Endsinger:663D:/
+559.0 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:663E:/
+563.5 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:6644:/
+576.7 "Elegeia" sync / 1[56]:[^:]*:The Endsinger:662C:/
+
+# Add spawns
+1000.0 "Planetes" sync / 1[56]:[^:]*:The Endsinger:6B58:/ window 1000,0
+1000.0 "--untargetable--"
+1014.3 "--targetable--"
+1023.4 "Meteor Radiant" sync / 1[56]:[^:]*:Kakodaimon:6656:/
+1035.2 "Crash" sync / 1[56]:[^:]*:Kakodaimon:6657:/
+1040.2 "Crash" sync / 1[56]:[^:]*:Kakodaimon:6657:/
+1045.2 "Crash" sync / 1[56]:[^:]*:Kakodaimon:6657:/
+1050.2 "Crash" sync / 1[56]:[^:]*:Kakodaimon:6657:/
+1056.6 "Meteor Outburst" sync / 1[56]:[^:]*:Kakodaimon:665C:/
+1067.8 "Katastrophe" sync / 1[56]:[^:]*:Kakodaimon:665B:/ window 0,30
+1078.9 "Katastrophe" sync / 1[56]:[^:]*:Kakodaimon:665B:/
+1090.0 "Katastrophe" sync / 1[56]:[^:]*:Kakodaimon:665B:/
```

Comment:
What I would normally do for something like this is to have 4-6 lines written out, and then jump from the second to the first. So it might look like this:

```
1067.8 "Katastrophe" sync / 1[56]:[^:]*:Kakodaimon:665B:/
1078.9 "Katastrophe" sync / 1[56]:[^:]*:Kakodaimon:665B:/ jump 1067.8
1090.0 "Katastrophe"
1101.1 "Katastrophe"
1112.2 "Katastrophe"
1123.3 "Katastrophe"
```

Technically we can probably get by with just four lines here, since 30s in the future is the max displayed normally, but I usually just make sure unreachable lookahead sections have at least six lines, including the jump line.